src/dev/flang/ast/Call.java

```diff
@@ -821,14 +837,23 @@ private boolean loadCalledFeatureUnlessTargetVoid(Resolution res, AbstractFeatur
                 else
                   {
                     var calledName = FeatureName.get(_name, _actuals.size());
-                    AstErrors.calledFeatureNotFound(this,
-                                                    calledName,
-                                                    tf,
-                                                    _target,
-                                                    FeatureAndOuter.findExactOrCandidate(fos,
-                                                                                        (FeatureName fn) -> false,
-                                                                                        (AbstractFeature f) -> f.featureName().equalsBaseName(calledName)),
-                                                    hiddenCandidates(res, thiz, tf, calledName));
+                    if (this._targetOf_forErrorSolutions != null
+                        && this._targetOf_forErrorSolutions._name.startsWith("infix ->")
+                        && this._targetOf_forErrorSolutions._name.length() > "infix ->".length())
+                      {
+                        AstErrors.usingLambdaOpertatorAsPrefixInOperator(this._targetOf_forErrorSolutions.pos());
+                      }
+                    else
+                      {
+                        AstErrors.calledFeatureNotFound(this,
+                                                        calledName,
+                                                        tf,
+                                                        _target,
```

Comment:
I would still report the `calledFeatureNotFound` error since that might still be the problem in case of a valid use of an infix operator like `->>`, but add the suggested solution only in this case.